src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -37,8 +38,95 @@ class LLDPLocalChassis(int, Enum):
     lldp_loc_chassis_id = 2
     lldp_loc_sys_name = 3
     lldp_loc_sys_desc = 4
-    # *lldp_rem_sys_cap_supported = 5
-    # *lldp_rem_sys_cap_enabled = 6
+    lldp_loc_sys_cap_supported = 5
+    lldp_loc_sys_cap_enabled = 6
+
+
+class ManAddrConst(Enum):
+    man_addr_if_id = 0
+    """
+    Reference [RFC2453][RFC2677][RFC2858]
+    Address Family Numbers
+    1	IP (IP version 4)
+    2	IP6 (IP version 6)
+    """
+    man_addr_subtype_ipv4 = 1
+    man_addr_subtype_ipv6 = 2
+    """
+    The enumeration 'ifIndex(2)' represents interface identifier
+    based on the ifIndex MIB object.
+    """
+    man_addr_if_subtype = 2
+    """
+    "The total length of the management address subtype and the
+    management address fields in LLDPDUs transmitted by the
+    local LLDP agent.
+    """
+    man_addr_len = 5
+    """
+    "The OID value used to identify the type of hardware component
+    or protocol entity associated with the management address
+    advertised by the local system agent."
+    """
+    man_addr_oid = (1, 3, 6, 1, 2, 1, 2, 2, 1, 1)
+
+
+def poll_lldp_entry_updates(pubsub):
+    msg = pubsub.get_message()
+
+    if not msg:
+        return None, None, None
+
+    lldp_entry = msg["channel"].split(b":")[-1].decode()
+    data = msg['data']
+
+    # extract interface name
+    interface = lldp_entry.split('|')[-1]
+
+    # get interface from interface name
+    if_index = port_util.get_index_from_str(interface)
+
+    if if_index is None:
+        # interface name invalid, skip this entry
+        logger.warning("Invalid interface name in {} in APP_DB, skipping"
+                       .format(lldp_entry))
+        return None, None, None
+    return data, interface, if_index
+
+
+class LLDPLocalSystemDataUpdater(MIBUpdater):
+    def __init__(self):
+        super().__init__()
+
+        self.db_conn = mibs.init_db()
+        self.loc_chassis_data = {}
+
+    def reinit_data(self):
+        """
+        Subclass update data routine.
+        """
+        # establish connection to application database.
+        self.db_conn.connect(mibs.APPL_DB)
+        self.loc_chassis_data = self.db_conn.get_all(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE)
+
+    def update_data(self):
+        """
+        Avoid NotImplementedError
+        The data is mostly static, reinit it once a minute is enough.
+        """
+        pass
+
+    def table_lookup(self, table_name):
+        try:
+            _table_name = bytes(getattr(table_name, 'name', table_name), 'utf-8')
+            return self.loc_chassis_data[_table_name]
+        except KeyError as e:
+            mibs.logger.warning(" 0 - b'LOC_CHASSIS' missing attribute '{}'.".format(e))
+            return None
+
+    def table_lookup_integer(self, table_name):
+        subtype_str = self.table_lookup(table_name)
+        return int(subtype_str) if subtype_str is not None else None
```

Comment:
>None [](start = 64, length = 4)

None is not an int. Is there an safe fallback value?

Reply:
The function is used to get ChassisIdSubtype, which is expected to be 4(mac address). I could hardcode it here, but i don't see the point.
Returning None here will result in END_OF_MIB, which is not that bad. If something is missing from LOC_CHASSIS table, table_lookup() has a log for that. 
What is the point of having fallback value for ChassisIdSubtype, if probably will also be missing the ChassisId itself? They are provided by the same flow. Looking up ChassisId will return the same None if misssing.

Reply:
@qiluo-msft do you insist on more changes or agree with my previous comment?